model/bindings.go

```diff
@@ -72,31 +66,36 @@ func CreateBindingsRancher(pathToYml string) BindingProperty {
 		log.Errorf("Error during Unmarshal  for rancher-compose: %v\n", err)
 	} else {
 		rawConfigRancher, err := preprocess.PreprocessServiceMap(rawConfigRancher)
-		utils.Convert(rawConfigRancher, &rancherComposeMap)
 		if err != nil {
 			fmt.Printf("Error")
 		} else {
-			keys := reflect.ValueOf(rancherComposeMap).MapKeys()
+			keys := reflect.ValueOf(rawConfigRancher).MapKeys()
 
 			for _, key := range keys {
 				if key.String() != ".catalog" {
 					newServiceBinding := ServiceBinding{}
 
 					newServiceBinding.Labels = mapLabel{}
-					newServiceBinding.Ports = []string{}
+					newServiceBinding.Ports = portArray{}
+					newServiceBinding.Scale = 0
+
+					if rawConfigRancher[key.String()]["scale"] != nil {
+						newServiceBinding.Scale = rawConfigRancher[key.String()]["scale"]
+					}
 
-					newServiceBinding.Scale = rancherComposeMap[key.String()].Scale
-					if configMap[key.String()].Labels != nil {
-						newServiceBinding.Labels = configMap[key.String()].Labels
+					if rawConfigDocker[key.String()]["labels"] != nil {
+						for k, v := range rawConfigDocker[key.String()]["labels"].(map[interface{}]interface{}) {
```

Comment:
Use `utils.Convert` to change `rawConfigDocker[key.String()]["labels"]` to a [`Sliceormap`](https://github.com/docker/libcompose/blob/master/yaml/types_yaml.go#L62). Then assign `newServiceBinding.Labels[k.(string)]` to the result of that conversion.